lib/postman/channel.rb

```diff
@@ -54,8 +54,10 @@ def queue
     end
 
     def queue_arguments
-      config = { 'x-dead-letter-exchange' => @deadletter_exchange }
+      config = {}
+      config['x-dead-letter-exchange'] = @deadletter_exchange
       config['x-message-ttl'] = @ttl if @ttl.present?
+      config['durable'] = true
       config
```

Comment:
More changes inspired by comments in the Readme. This should just reduce the number of tweaks needed to get things up and running.